### ISSUE TITLE
Weight initialization 參考網頁

### ISSUE THREAD

**Issue by yyrkoon27** [yyrkoon27/2017-fall-DL-training-program-MM#3]
https://discuss.pytorch.org/t/weight-initilzation/157/8
Please search "A less Lua way of doing that ..."

**Comment by yyrkoon27** [comment]
另看到一種做法. 
https://github.com/pytorch/vision/blob/master/torchvision/models/squeezenet.py
它是針對最後一個 convolution, 做 zero mean, small std 的 normalization,
至於其他的 convolution, 則做 kaiming_uniform
```
for m in self.modules():
    if isinstance(m, nn.Conv2d):
        if m is final_conv:
            init.normal(m.weight.data, mean=0.0, std=0.01)
        else:
            init.kaiming_uniform(m.weight.data)
        if m.bias is not None:
            m.bias.data.zero_()
```
